testing/unit/task_test.ts

```diff
@@ -0,0 +1,32 @@
+// Copyright 2018-2024 the Deno authors. All rights reserved. MIT license.
+import { test } from "checkin:testing";
+
+const { op_task_submit } = Deno.core.ops;
+
+test(async function testTaskSubmit1() {
+  const { promise, resolve } = Promise.withResolvers();
+  op_task_submit(() => {
+    resolve(undefined);
+  });
+  await promise;
+});
+
+test(async function testTaskSubmit2() {
+  for (let i = 0; i < 2; i++) {
+    const { promise, resolve } = Promise.withResolvers();
+    op_task_submit(() => {
+      resolve(undefined);
+    });
+    await promise;
+  }
+});
+
+test(async function testTaskSubmit3() {
+  for (let i = 0; i < 3; i++) {
+    const { promise, resolve } = Promise.withResolvers();
+    op_task_submit(() => {
+      resolve(undefined);
+    });
+    await promise;
+  }
+});
```

Comment:
Does it still work fine if the loop has 100 or 1000 iterations?

Reply:
It does -- I added a 100 iteration test "just in case"